.github/actions/run-on-image/action.yaml

```diff
@@ -45,6 +45,7 @@ runs:
     - run: >-
         docker run
         --user $(id -u):$(id -g)
+        --env npm_config_cache=/tmp/.npm
```

Comment:
otherwise `/.npm` is used, where the default user does not have permission to write by default.